geode-core/src/main/java/org/apache/geode/internal/util/PasswordUtil.java

```diff
@@ -44,72 +42,29 @@
   private static byte[] init = "string".getBytes();
 
   /**
-   * Encrypts a password string
-   * 
-   * @param password String to be encrypted.
-   * @return String encrypted String
-   */
-  public static String encrypt(String password) {
-    return encrypt(password, true);
-  }
-
-  /**
-   * 
-   * @param password String to be encrypted
-   * @param echo if true prints result to system.out
-   * @return String encrypted String
+   * Decrypts an encrypted password string.
+   *
+   * @param password String to be decrypted
+   * @return String decrypted String
    */
-  public static String encrypt(String password, boolean echo) {
-    String encryptedString = null;
+  @Deprecated
+  public static String decrypt(String password) {
+    String toDecrypt;
+    if (password.startsWith("encrypted(") && password.endsWith(")"))
```

Comment:
I don't think it gets caught by spotless, but officially we prefer to wrap every `if` and `else` in curly braces, even when they're just one-liners.